work_queue/src/network_function.py

```diff
@@ -50,18 +52,17 @@ def main():
             if event_size:
```

Comment:
Put this `if event_size` inside a  `try ... except ...` block, so that the coprocessor does not die if something goes wrong reading the response, or forking, etc.